gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -71,6 +71,8 @@ export MNAIO_SSH="ssh -ttt -oStrictHostKeyChecking=no root@infra1"
 # place variable in file to be sourced by parent calling script 'run'
 export MNAIO_VAR_FILE="${MNAIO_VAR_FILE:-/tmp/mnaio_vars}"
 echo "export MNAIO_SSH=\"${MNAIO_SSH}\"" > "${MNAIO_VAR_FILE}"
+echo "export RPC_RELEASE=\"${RPC_RELEASE}\"" > "${MNAIO_VAR_FILE}"
```

Comment:
This line needs to append to the `MNAIO_VAR_FILE`. So, the `>>` redirection operator needs to be used. Otherwise each call will overwrite the file with a single line of new content.

Line should read as follows.
```
echo "export RPC_RELEASE=\"${RPC_RELEASE}\"" >> "${MNAIO_VAR_FILE}"
```